config/crds/tenancy.kcp.dev_clusterworkspacetypes.yaml

```diff
@@ -53,9 +53,24 @@ spec:
                   description: ClusterWorkspaceInitializer is a unique string corresponding
                     to a cluster workspace initialization controller for the given
                     type of workspaces.
-                  maxLength: 32
-                  pattern: ^([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*/)?(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$
-                  type: string
+                  properties:
+                    name:
+                      description: name is a unique string corresponding to a cluster
+                        workspace initialization controller for the given type of
+                        workspaces.
+                      maxLength: 32
+                      pattern: ^([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*/)?(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$
+                      type: string
+                    path:
+                      description: path is an absolute reference to the workspace
+                        that owns this initializer, e.g. root:org:ws. The workspace
+                        must be some ancestor or a child of some ancestor.
```

Comment:
Copy and pasta text? For the type I would expect that it is just the clusterName of this object. I.e. we only have to store it on the ClusterWorkspace because there the initializers can come from different workspaces.

Reply:
If we have one central API Binding application controller that's simply configured differently per ClusterWorkspaceType, this invariant is already broken/